doc/source/data-ingest.rst

```diff
@@ -0,0 +1,140 @@
+
+Conduce needs a few key pieces of information in order to visualize your data in space and time, those are:
+
+**Identity**
+   Identities distinguish one entity from another.  An entity exists in Conduce as one more more samples.  All samples that describe the same entity share an identity.
+**Location**
+   An entities location describes where in space the entity exists.  Locations are defined in units according to the coordinate system in which they are displayed. (Latitude and Longitude on a world map.)
+**Time**
+   Time defines the moment in time at which the rest of the information describing an entity is valid.
+
+With this information Conduce can build a representation of an object that exists in space and time.  Entity definitions can be more complex, for more information on how to build entities please read :doc:`conduce-entities`.
+
+--------
+Datasets
+--------
+
+Datasets are the containers that hold entities.  A dataset may be thought of as a table.  Each entity in a dataset is similar in nature, but has unique properties.  In order to represent and entity in Conduce, its sample data must be ingested into a dataset.  In order to perform any API operations it is recommended that you generate an API key.  (See :doc:`api-key-creation` for more information.)  The first step in ingesting data is acquiring a dataset ID.  If you are ingesting data into a new dataset you'll need to create the dataset  before entities can be ingested.
+
+:py:func:`api.create_dataset`::
+
+    world_cities_dataset_id = conduce.api.create_dataset('world cities', host='app.conduce.com', api-key='00000000-0000-0000-0000-000000000000')
+
+The dataset ID returned is a UUID.  It will be passed as a function parameter to the :py:func:`api.ingest_samples` function.
+
+If you want to ingest data into an existing dataset you will need to acquire its dataset ID.  Datasets are Conduce resources.  All resources have an ID and can be queried with the resource API.  The Conduce Python API provides convenience methods for listing datasets
+
+:py:func:`api.list_datasets`::
+
+    datasets = conduce.api.list_datasets(host='app.conduce.com', api-key='00000000-0000-0000-0000-000000000000')
+
+This call returns a list of dictionaries, each contains information about a particular dataset::
+
+    {
+       "name": "US Cities",
+       "tags": [],
+       "create_time": 1506544998452,
+       "version": 0,
+       "mime": "application/json",
+       "modify_time": 1508869186664,
+       "revision": 4,
+       "type": "DATASET",
+       "id": "a262342b-22d2-4368-4698-cf75b62b7cb3",
+       "size": 2
+     }
+
+This dataset object contains information to help identify a dataset.  The fields ``name`` and ``tags``, along with ``create_time`` and ``modify_time`` are particularly useful for identifying datasets.  The :py:func:`api.find_datasets` function can be used to list only datasets that match certain search criteria.
```

Comment:
it might be worth reiterating that only the uuid is truly unique - there can be 2 datasets with the same name, tag, & less likely so, the same create_time & modify_time.

Reply:
re-tagging this - not sure if it was caught & is now hidden.

Reply:
I considered it.  I want to keep the focus on ingest.  Details of resources and their metadata seem off topic.  I did want to provide some information to help a user employ the commands the document recommends.